editions/de-AT-server/tiddlywiki.info

```diff
@@ -2,7 +2,8 @@
 	"description": "Server configuration of the German (Österreich) edition",
 	"plugins": [
 		"tiddlywiki/tiddlyweb",
-		"tiddlywiki/filesystem"
+		"tiddlywiki/filesystem",
+		"tiddlywiki/highlight"
```

Comment:
The motivation for @araname's changes is to make the language editions more consistent, which I think is desirable.

How does the addition of the highlight plugin affect your workflow? Is it just the general size issue?

Perhaps we should remove the language `*-server` editions, and encourage the editors of language editions to create their own server edition folders outside the repo that point to the associated language edition within the repo? That way each language editor could arrange their server edition as they like it for general editing, while keeping the content of the main repo consistent between editions?